pipeline-model-api/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/ast/ModelASTStage.java

```diff
@@ -21,6 +21,7 @@
     private ModelASTWhen when;
     private ModelASTTools tools;
     private ModelASTEnvironment environment;
+    private Boolean failFast;
```

Comment:
So we should not provide the ability to say `failFast = env.BRANCH != "master"` ? >:)